index.js

```diff
@@ -29,7 +29,7 @@ module.exports = function (collection, callback) {
     }
 
     if (isPolygon || isMultiPolygon) {
-      if (isPolygon) coords = [coords]
```

Comment:
Turns out this was needed. I added a complex polygon + multipolygon test that failed until I put this back. The reason it was passing before is that your multipolygon example below is actually just a simple polygon. I replaced it with the mutlipolygon example from http://geojson.org/geojson-spec.html#id7 and did some fixing and everything works now.